docs/Developers/Profile_Bears.rst

```diff
@@ -0,0 +1,74 @@
+Profile Bears
+=============
+
+This document provide the overview of coala's Profiling Interface. The
+Profiling Interface will provide an ability to Profile the Bear's code to
+optimize its performance.
+
+The profiler will start by profiling the run() method of bears because it
+consumes most of the bears time. So, this is part where bear writer will spend
+time, as rest of the part like loading the files, collecting the settings,
+etc. are done by coala itself.
+
+.. note::
```

Comment:
shouldn't be there another newline here? Not sure if RST requires that

Reply:
Both will work fine IMO.